pkgs/development/interpreters/python/hooks/pip-install-hook.sh

```diff
@@ -22,3 +25,20 @@ if [ -z "${dontUsePipInstall-}" ] && [ -z "${installPhase-}" ]; then
     echo "Using pipInstallPhase"
     installPhase=pipInstallPhase
 fi
+
+
+pipAuditTmpdir() {
```

Comment:
There is also a tool called pip-audit which makes this a bit confusing

Reply:
I was trying to follow the terminology from here because it's mostly extending that safety net: https://github.com/NixOS/nixpkgs/blob/3fa49138c9e24841bc68f73e9fb80998d886abab/pkgs/build-support/setup-hooks/audit-tmpdir.sh#L12

But just tell me what name you prefer, I can change it, no problem.